### ISSUE TITLE
openXsensor2.5  not compatible with latest configurator 3.0

### ISSUE THREAD

**Issue by Ryanf55** [openXsensor/openXsensor#23]
<img width="205" alt="varioconfigbad" src="https://cloud.githubusercontent.com/assets/25047695/21834740/f1d18ef6-d774-11e6-9eba-a31402a5371a.PNG">

I got this error when trying to write my config. I have downloaded the latest OX sensor and configurator from the downloads section. 

Thanks!

**Comment by davxlw** [comment]
Unfortunately, there is not an openXsensor v3.0 official release in the download section :(
But you can get it through this link: https://github.com/openXsensor/openXsensor/archive/OpenXsensor_Multiplex_v3.0.zip

**Comment by Ryanf55** [comment]
Thanks! 
I downloaded that file and used the configurator in the downloads section. 
<img width="190" alt="error2" src="https://cloud.githubusercontent.com/assets/25047695/22583827/3579d340-e9ac-11e6-95ab-3500d0cfe0e0.PNG">


I can't get the variometer working. I have used/sold KK variometers, and wanted to try the OpenTX version. It doesn't give me any data when using the configurator. 

However, if I flash the arduino default sketch, it does work as a variometer, but obviously is not configurable. 

Here's my config file, using a D4R II receiver, sending data from arduino D4 to RX pin of the D4R ii receiver. 

Thanks!
